supply/lib/supply/setup.rb

```diff
@@ -12,8 +12,7 @@ def perform_download
 
       client.listings.each do |listing|
         store_metadata(listing)
-        create_screenshots_folder(listing)
```

Comment:
What happens if there are no screenshots available at Google Play - are the folders created correctly anyway now with those changes?

Reply:
https://github.com/fastlane/fastlane/pull/14322/commits/379316c2ce3f1663a1ba626b1682d270760e005f

Reply:
Yes :)

> What happens if there are no screenshots available at Google Play - are the folders created correctly anyway now with those changes?

Yes :) This line ensures folders are created before hand - https://github.com/fastlane/fastlane/blob/379316c2ce3f1663a1ba626b1682d270760e005f/supply/lib/supply/setup.rb#L63

Reply:
Doesn't this get skipped when there are no screenshots via https://github.com/fastlane/fastlane/pull/14322/files#diff-674b785c2e89b32d433060b131c0cf28R54?

Reply:
Yes! If there are no images returned (for example, `tvBanner` in most cases) doesn't have images. So there's no need to create its folder. 

I think if the user uploads images for `tvBanner`, then next time someone tries to download the images, `tvBanner` will be created as well!

Reply:
Yes, for the images that's correct - but not for the screenshots. There the folder always has to exist (at least that is how I red the old code).

Reply:
https://github.com/fastlane/fastlane/pull/14322/commits/42e3768b7d59364229c9022add8a0b0809e0b8a1